parts/linux/cloud-init/artifacts/components.json

```diff
@@ -650,8 +650,7 @@
             "versionsV2": [
               {
                 "renovateTag": "name=moby-containerd, os=ubuntu, release=20.04",
-                "latestVersion": "1.7.20"
-              }
+                "latestVersion": "1.7.22-ubuntu20.04u1"}
```

Comment:
wonder why this hasn't been picked up for 2204

Reply:
I was also aware of this. I was wondering if the PR that Renovate can create reach a limit. (or maybe not). Will follow up with this.